openmetadata-ui/src/main/resources/ui/src/components/Customization/GenericWidget/GenericWidget.tsx

```diff
@@ -0,0 +1,664 @@
+          style: {},
+          fullyQualifiedName: 'Business Department.Marketing',
+          synonyms: ['Sell', 'Retails'],
+          glossary: {
+            id: 'dae534b6-f5d1-4fc7-9ddf-0d1ec9df5c7e',
+            type: 'glossary',
+            name: 'Business Department',
+            fullyQualifiedName: 'Business Department',
+            description:
+              'Businesses often have several departments that perform unique functions, allowing them to operate efficiently and successfully.',
+            displayName: 'Business Department',
+            deleted: false,
+            href: 'http://sandbox-beta.open-metadata.org/api/v1/glossaries/dae534b6-f5d1-4fc7-9ddf-0d1ec9df5c7e',
+          },
+          references: [],
+          version: 0.2,
+          updatedAt: 1700558309238,
+          updatedBy: 'shailesh',
+          href: 'http://sandbox-beta.open-metadata.org/api/v1/glossaryTerms/e866ee75-711a-4649-968d-3ea889bd75b8',
+          owners: [],
+
+          status: 'Rejected',
+          deleted: false,
+
+          mutuallyExclusive: false,
+          childrenCount: 1,
+        },
+        {
+          id: '288cfb46-a4c2-45a4-9dc0-321eac165812',
+          name: 'test_business_term',
+          displayName: 'Test Business Term',
+          description: 'this is test_business_term',
+          fullyQualifiedName: 'Business Department.test_business_term',
+          version: 0.2,
+          updatedAt: 1728547870161,
+          updatedBy: 'karan',
+          href: 'http://sandbox-beta.open-metadata.org/api/v1/glossaryTerms/288cfb46-a4c2-45a4-9dc0-321eac165812',
+          owners: [],
+          deleted: false,
+          mutuallyExclusive: false,
+        },
+      ] as ModifiedGlossary[]);
+    }
+
+    return () => setGlossaryChildTerms([]);
+  }, [props.widgetKey]);
+
+  const widgetName = startCase(props.widgetKey.replace('KnowledgePanel.', ''));
+
+  const cardContent = useMemo(() => {
+    if (
+      props.widgetKey.startsWith(DetailPageWidgetKeys.GLOSSARY_TERMS) ||
+      props.widgetKey.startsWith(GlossaryTermDetailPageWidgetKeys.RELATED_TERMS)
+    ) {
+      return (
+        <TagsViewer
+          displayType={DisplayType.READ_MORE}
+          showNoDataPlaceholder={false}
+          tagType={TagSource.Glossary}
+          tags={[
+            {
+              tagFQN: 'BusinessGlossary.Purchase',
+              source: TagSource.Glossary,
+              name: 'Purchase',
+            },
+            {
+              tagFQN: 'Person.BankNumber',
+              source: TagSource.Glossary,
+              name: 'BankNumber',
+            },
+            {
+              tagFQN: 'Hospitality.Guest Type',
+              source: TagSource.Glossary,
+              name: 'Guest Type',
+            },
+            {
+              tagFQN: 'Financial Services',
+              source: TagSource.Glossary,
+              name: 'Auto Loan',
+            },
+          ]}
```

Comment:
Why are we passing hard-coded tags list?

Reply:
It will be seen at the time of customization, unlike the landing page, we don't have any API to populate data here, so we need to hardcode it